Upstatement/ups-public-library
# Add Everett trial fonts and comment out unused JS for now

## Changes

Trying to start with a cleaner slate

- Added trial fonts
- Commented out some JS for now to prevent errors in the console

## Steps to test

- Disable the everett font on your computer
- Open up the index page and make sure the font loads properly
- Make sure there's no errors in the console
